app/controllers/api/v1/organisations_users_controller.rb

```diff
@@ -47,6 +48,10 @@ def show
         render json: @organisations_user, serializer: serializer
       end
 
+      def user_status
+        render json: { status: OrganisationsUser.all_status }, status: 200
+      end
+
```

Comment:
Isn't it a bit overkill to create an endpoint just for a few strings ? Maybe using a `goodcity_setting` record is more appropriate ?

You could use an initializer to create it perhaps ?

Reply:
I'm just thinking out loud. Any thoughts @steveyken  ?